src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpAgoraDAO.scala

```diff
@@ -0,0 +1,28 @@
+package org.broadinstitute.dsde.firecloud.dataaccess
+
+import akka.actor.ActorSystem
+import org.broadinstitute.dsde.firecloud.FireCloudConfig
+import org.broadinstitute.dsde.firecloud.model.MethodRepository.AgoraPermission
+import org.broadinstitute.dsde.firecloud.model.ModelJsonProtocol._
+import org.broadinstitute.dsde.firecloud.model.UserInfo
+import org.broadinstitute.dsde.firecloud.utils.RestJsonClient
+import spray.client.pipelining._
+import spray.httpx.SprayJsonSupport._
+import spray.json.DefaultJsonProtocol._
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class HttpAgoraDAO(implicit val system: ActorSystem, implicit val executionContext: ExecutionContext)
+  extends AgoraDAO with RestJsonClient {
+
+  private def getNamespaceUrl(ns: String, entity: String): String = {
+    s"${FireCloudConfig.Agora.authUrl}/$entity/$ns/permissions"
+  }
```

Comment:
On second thought, loading a config would mean having to deconstruct that config the same way we're doing it in FireCloudConfig, but again in the dao class. I can see the case for constructing it with specific elements, but not the whole config.